trim.production.js

```diff
@@ -13,48 +19,79 @@ var token;
  * @param {String} recordNo The TRIM RecordNo of the new document, if created.
  */
 
-
 /**
- * @param {String} title Record Title
- * @param {String} container Name of the container to upload to
- * @param {String} extension without the period, e.g. "pdf" or "png" but not ".png"
- * @param {String} fileData base64 encoding of the date without the dateURL prefix stuff
- * @param {String[]} alternativeContainers Names of containers to link the file to in addition to the "container" param
- * @param {createRecordCallback} callback
+ * Uploads a Document to TRIM
+ * @param {Object}    data
+ * @param {String}    data.title
+ * @param {String}    data.container Primary container to upload to
+ * @param {String}    data.extension Omit the period, e.g. "pdf" or "png" but not ".png"
+ * @param {String}    data.fileData Base64 String without the prefix
+ * @param {String[]}  data.alternativeContainers A list of additional TRIM containers to upload to
+ * @param {Boolean}   data.privacyLevel public=1, private=2 -- higher numbers escalate privacy level
+ * @param {createRecordCallback}  callback
  */
-function createRecord (title, container, extension, fileData, alternativeContainers, callback) {
-  if (typeof alternativeContainers === 'function') {
-    throw new Error('Migration Error: a new parameter was added: alternativeContainers')
+function createRecord(data, callback) {
+  if (arguments.length === 5) {
+    // Compatibility with usage before the alternativeContainers parameter was added
+    debug('WARNING: The createRecord arguments have changed to accept a single data parameter instead of multiple')
```

Comment:
Yeah probably. It was a console.log until it was flooding the screen with logs when running the tests. Maybe it's better to set a "testing" state or something and only console.log if you're not testing.